### ISSUE TITLE
Remove all the deprecated attributes

### ISSUE THREAD

**Issue by eric-wieser** [pygae/galgebra#285]
After we do a 0.5.0 release, we should look at removing all of the deprecated properties and methods from the source code, so that 0.6.0 no longer has them.

Grepping for `stacklevel=` seems to find most (all?) of these. As of writing, there are 26 of these members in the source so far, and removing them would save about 200 lines of code. We should only apply this to deprecations which existed by the time of the 0.5.0 release.

**Comment by utensil** [comment]
Should we continue to finish #196 in 0.6.0 and do this in the next release? So we can tell the user to try 0.6.0 first to see what should be used instead and then move to newer versions without a clue.

**Comment by eric-wieser** [comment]
My original plan was:

* In 0.6.0, remove the things we already deprecated in 0.5.0
* In 0.6.0, deprecate any more that are easy to get rid of
* In 0.7.0, deprecate the ones from the point above.

However, another option would be:

* In 0.6.0, deprecate any more that are easy to get rid of
* In 1.0.0, remove all of them.

**Comment by utensil** [comment]
Yeah I prefer the latter as the deprecated methods are actually no longer in our ways and it would provide a smoother migration process by this middle point ("So we can tell the user to try 0.6.0 first to see what should be used instead and then move to newer versions without a clue.").

**Comment by utensil** [comment]
I still imagine that there is quite some code written in the past since I've read quite some papers claiming the author used GAlgebra to verify some identities, although I haven't seen them coming and the code is likely not open source, they might still need to migrate to the new version one day.
